composer.json

```diff
@@ -95,7 +95,8 @@
     "scheb/two-factor-bundle": "^3.26",
     "phpoffice/phpspreadsheet": "^1.9",
     "html2text/html2text": "^4.2.1",
-    "webmozarts/console-parallelization": "^1.1.0"
+    "webmozarts/console-parallelization": "^1.1.0",
+    "brick/varexporter": "^0.3.2"
```

Comment:
This should be an optional dependency, I think, since it's only used in one place and on demand.